src/site/content/en/blog/richer-offline-w-periodic-background-sync/index.md

```diff
@@ -0,0 +1,458 @@
+</td>
+<td markdown="block">
+<strong><a href="https://developers.chrome.com/origintrials/#/view_trial/4048736065006075905">Started in Chrome 77<a></strong><br/>
+Expected to run through Chrome 80
+</td>
+</tr>
+<tr>
+<td markdown="block">
+5. Launch
+</td>
+<td markdown="block">
+Not started
+</td>
+</tr>
+</table>
+
+## Try it
+
+You can try periodic background sync with our [live demo
+app](https://webplatformapis.com/periodic_sync/periodicSync_improved.html).
+Before using it, make sure that:
+
+* You're using Chrome 77 or later.
+* You
+  [install](https://developers.google.com/web/fundamentals/app-install-banners/)
+  the web app before enabling periodic background sync. (The demo app's
+  author already took the step of signing up for the origin trial.)
+
+### Register for the origin trial {: #ot }
+
+{% include 'content/origin-trials.njk' %}
+
+{% include 'content/origin-trial-register.njk' %}
+
+## Concepts and usage
+
+Periodic background sync lets you show fresh content when a progessive web app
+or service worker-backed page is launched. It does this by downloading data in
+the background when the app or page is not being used. Without it, the app's
+content will refresh after launch, leading to content changing while the user is
+viewing it. Worse yet, the app may only show a content spinner.
+
+Without periodic background sync, web apps must use alternative methods to
+download data. A common example is using a push notification to wake a service
+worker. The user is interrupted by a message such as 'new data available'.
+Updating the data is essentially a side effect. You still have the option of
+using push notifications for truly important updates, such as significant
+breaking news.
+
+Periodic background sync is easily confused with background sync. Though they
+have similar names, their use cases are different. Among other things,
+background sync is most commonly used for resending data to a server when a
+previous request has failed.
+
+### Getting this right
+
+We are putting periodic background sync through a trial period so that you can
+help us make sure that we get it right. This section explains some of the design
+decisions we took to make this feature as helpful as possible.
+
+The first design decision we made is that a web app can only use periodic
+background sync after a person has installed it on their device, and has
+launched it as a distinct application. Periodic background sync is not available
+in the context of a regular tab in Chrome.
+
+Furthermore, since we don't want unused or seldom used web apps to gratuitously
+consume battery or data, we designed periodic background sync such that
+developers will have to earn it by providing value to their users. Concretely,
+we are using a site engagement score to determine if and how often periodic
+background syncs can happen for a given web app. In other words, a
+`periodicsync` event won't be fired at all unless the engagement score is
```

Comment:
`periodicsync` hasn't been introduced yet.

Reply:
One of the reasons I wanted to rewrite this article. Unfortunately this is text we can't change.